_posts/2018-10-26-schedule-sneak-peek-rome.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Schedule sneak peek"
+socialImageSrc: "/assets/posts/rome/twirf-rustbridge.jpg"
+authors:
+- dns2utf8
+---
+
+The schedule is long awaited and it is the one thing the organisation has to make sure is interesting and digestible.
+Plus the topics should be varied.
+
+So where is the schedule? Since the talks are already [published on the website](https://rome.rustfest.eu/talks/), how hard can it be?
+Well here is the current draft:
+
+| Time  | Talk or Event      |
+|-------|--------------------|
+| 08:30 | Registration opens |
+| 09:00 | Introduction from local orga team + sponsors |
+| 09:30 | Opening keynote |
+| ...   | ... |
+| 10:45 | Coffee break |
+| ...   | ... |
+| 12:30 | Lunch break |
+| 14:00 | Afternoon talks |
+| ...   | ... |
+| 15:05 | Coffee break |
+| ...   | ... |
+| 16:55 | Closing keynote |
+| 17:40 | Tips and final words from the team |
+| 17:55 | Family photo |
+| 18:00 | Doors close |
+
+You might be wondering "did they just write times like `:40`?"
+Yes we did!
+There are some Swiss people in the orga and to maximize the amount of talks we will be punctual and on time.
```

Comment:
Thanks to the Swiss people for keeping at bay the Italians :-))
(joking)

Reply:
Hope people get the wink without an emojy ;)